Geometry/VeryForwardGeometryBuilder/src/DetGeomDesc.cc

```diff
@@ -11,66 +11,113 @@
 
 #include "Geometry/VeryForwardGeometryBuilder/interface/DetGeomDesc.h"
 #include "Geometry/VeryForwardGeometryBuilder/interface/CTPPSDDDNames.h"
+#include "CondFormats/GeometryObjects/interface/PDetGeomDesc.h"
```

Comment:
I think the following now need to be added to `Geometry/VeryForwardGeometryBuilder/BuildFile.xml`:
`CondFormats/GeometryObjects`
`DetectorDescription/Core`
`DetectorDescription/DDCMS`